week-2/E-conditionals/exercise.js

```diff
@@ -11,9 +11,15 @@ var role = "mentor";
 var name2 = "Harun";
 var role2 = "student";
 
+if ("student") {
```

Comment:
Double check this one. I know you did this one a while ago – can you spot the mistake now?

Reply:
Again, I've uploaded a revision of this. Let me know if it was what you meant. Although I feel like I have a decent understandings of If statements I am not 100% sure what the question is asking me to do.